app/views/events/ranking.html.erb

```diff
@@ -0,0 +1,36 @@
+<div class="page-header">
+  <h1><%= t '.title' %></h1>
+</div>
+
+<table class="table table-striped">
+  <thead>
+  <tr>
+    <th><%= t '.rank' %></th>
+    <th><%= t '.name' %></th>
+    <th><%= t '.match_count' %></th>
+    <th><%= t '.won_count' %></th>
+    <th><%= t '.draw_count' %></th>
+    <th><%= t '.lost_count' %></th>
+    <th><%= t '.goals' %></th>
+    <th><%= t '.goals_against' %></th>
+    <th><%= t '.goals_difference' %></th>
+    <th><%= t '.points' %></th>
+  </tr>
+  </thead>
+  <tbody>
+    <% @ranking_entries.each do |entry| %>
+    <tr>
```

Comment:
Whole block could be replaced by something like this:
```
<tr>
  <% entry.values do |value| %>
    <td><%= value %></td>
  <% end %>
</tr>
```
Not sure what is more readable @bakoe @jonasnoki